src/main/java/com/strandls/integrator/services/impl/RuleFilterServiceImpl.java

```diff
@@ -810,7 +810,7 @@ public void bulkFilteringIn(HttpServletRequest request, Long userGroupId,
 						ugActivity.setUserGroupId(ugIbp.getId());
 						ugActivity.setUserGroupName(ugIbp.getName());
 						ugActivity.setWebAddress(ugIbp.getWebAddress());
-						ugActivity.setReason("Added Through Filter Rules");
+						ugActivity.setReason("via filter rules");
```

Comment:
could pls store the string in a variable and use it , instead of directly using the string